src/main/java/svc/data/citations/datasources/transformers/CourtIdTransformer.java

```diff
@@ -44,6 +45,23 @@
 			return null;
 		}
 	}
+	
+	public List<Long> getCourtIdsFromMunicipalityIds(List<Long>municipalityIds){
```

Comment:
Why did you choose to put this here? It's not transforming anything. This seems more appropriate in a `CourtManager` or something

Reply:
I put this here because this is very specific code used by the transformer.  The transformer does something similar for REJIS and TYLER (using the function "lookupCourtId"), but ITI needed this unique code.  So I place it here because of what uses it and it's function in the code is for the transformer.